graphql_server/channels/graphql_transport_ws.py

```diff
@@ -24,10 +24,10 @@ def __init__(
         self._ws = ws
 
     async def get_context(self) -> Any:
-        return await self._get_context(request=self._ws)
+        return await self._get_context(self._ws)
 
     async def get_root_value(self) -> Any:
-        return await self._get_root_value(request=self._ws)
+        return await self._get_root_value(self._ws)
```

Comment:
Removed this because [channels/graphql_ws.py](https://github.com/graphql-python/graphql-server/blob/v3/graphql_server/channels/graphql_ws.py#L27) doesn't specify the argument name either.